pkg/controller/util/cluster_util.go

```diff
@@ -32,10 +32,9 @@ import (
 
 const (
 	DefaultKubeFedSystemNamespace = "kube-federation-system"
-
-	KubeAPIQPS   = 20.0
-	KubeAPIBurst = 30
-	TokenKey     = "token"
+	KubeAPIQPS                    = 20.0
+	KubeAPIBurst                  = 30
+	TokenKey                      = "token"
```

Comment:
Please revert this unrelated change.

Reply:
This is related to the `go-fmt` issue. Let me paste the code as-is.